lib/api/core/models/security/role.js

```diff
@@ -330,20 +335,17 @@ function checkRestrictions(request, restrictedTo) {
  * @returns {Promise}
  */
 function executeClosure (role, kuzzle, path, actionRights, request) {
-  let
-    error,
-    argsDefinitions = {},
-    message;
+  let argsDefinitions = {};
   const pathstr = path.join('/');
 
   if (typeof actionRights.test !== 'string') {
-    error = errorsManager.get(
-      'parsing_rights_for_role',
+    return errorsManager.reject(
+      'security',
+      'role',
+      'closure_missing_test',
       role._id,
       pathstr,
       actionRights);
-    error.details = 'Missing or malformed "test" attribute (string required)';
```

Comment:
`details` doesn't exist in KuzzleError errors, it is thus not serialized, and so it's useless (also, adding this unexpected property to a class probably triggers a deopt from turbofan)